src/components/forms/PlanForm/index.tsx

```diff
@@ -127,6 +126,14 @@ export interface PlanFormProps {
   initialValues: PlanFormFields /** initial values for fields on the form */;
   jurisdictionLabel: string /** the label used for the jurisdiction selection */;
   redirectAfterAction: string /** the url to redirect to after form submission */;
+  renderLocationNames?: (
+    child?: (
+      locationName: string,
+      locationId: string,
+      index: number
+    ) => JSX.Element /** nested render content for each location name */
+  ) => JSX.Element;
+  /** a render prop that renders the plan's location names */
```

Comment:
@p-netm why the nested functions? Is it possible to flatten to a single func that just accepts those three params?

Reply:
Am afraid the other alternative would be to get rid of renderLocationNames props entirely and use the `PlanLocationNames` component from within `PlanForm`. having a nested renderProp is a consequence of passing the component responsible for rendering the locationNames as a prop.
@cKellyDesign @moshthepitt